frontend/src/metabase/query_builder/components/CompareAggregations/components/ColumnPicker/ColumnPicker.tsx

```diff
@@ -0,0 +1,76 @@
+import type { ComponentPropsWithoutRef } from "react";
+import { forwardRef, useCallback } from "react";
+import { t } from "ttag";
+
+import { Flex, MultiAutocomplete, Text } from "metabase/ui";
+
+import type { ColumnType } from "../../types";
+
+type ItemType = {
+  example: string;
+  label: string;
+  value: ColumnType;
+};
+
+interface Props {
+  value: ColumnType[];
+  onChange: (value: ColumnType[]) => void;
+}
+
+const COLUMN_OPTIONS: ItemType[] = [
+  {
+    example: "1826, 3004",
+    label: t`Previous value`,
+    value: "offset",
+  },
+  {
+    example: "+2.3%, -0.1%",
+    label: t`Percentage difference`,
+    value: "percent-diff-offset",
+  },
+  {
+    example: "+42, -3",
+    label: t`Value difference`,
+    value: "diff-offset",
+  },
+];
+
+export const ColumnPicker = ({ value, onChange }: Props) => {
+  const handleChange = useCallback(
+    (values: string[]) => {
+      onChange(values as ColumnType[]);
+    },
+    [onChange],
+  );
+
+  return (
+    <MultiAutocomplete
+      label={t`Columns to create`}
+      data={COLUMN_OPTIONS}
+      itemComponent={Item}
+      placeholder={t`Columns to create`}
+      rightSection={null}
+      shouldCreate={shouldCreate}
+      value={value}
+      onChange={handleChange}
+    />
+  );
+};
+
+const shouldCreate = () => false;
```

Comment:
If you don't need to create, maybe use `MultiSelect` instead?

Reply:
71e18de13ec0271c673c7bd7d1961c64ad0222ef